databricks/koalas/missing/frame.py

```diff
@@ -18,119 +18,121 @@
 
 
 def unsupported_function(method_name, deprecated=False, reason=""):
-    return _unsupported_function(class_name='pd.DataFrame', method_name=method_name,
-                                 deprecated=deprecated, reason=reason)
+    return _unsupported_function(
+        class_name="pd.DataFrame", method_name=method_name, deprecated=deprecated, reason=reason
+    )
 
 
 def unsupported_property(property_name, deprecated=False, reason=""):
-    return _unsupported_property(class_name='pd.DataFrame', property_name=property_name,
-                                 deprecated=deprecated, reason=reason)
+    return _unsupported_property(
+        class_name="pd.DataFrame", property_name=property_name, deprecated=deprecated, reason=reason
+    )
 
 
 class _MissingPandasLikeDataFrame(object):
 
     # Properties
-    axes = unsupported_property('axes')
-    iat = unsupported_property('iat')
-    ndim = unsupported_property('ndim')
+    axes = unsupported_property("axes")
+    iat = unsupported_property("iat")
+    ndim = unsupported_property("ndim")
 
     # Deprecated properties
-    blocks = unsupported_property('blocks', deprecated=True)
-    ftypes = unsupported_property('ftypes', deprecated=True)
-    is_copy = unsupported_property('is_copy', deprecated=True)
-    ix = unsupported_property('ix', deprecated=True)
+    blocks = unsupported_property("blocks", deprecated=True)
+    ftypes = unsupported_property("ftypes", deprecated=True)
+    is_copy = unsupported_property("is_copy", deprecated=True)
+    ix = unsupported_property("ix", deprecated=True)
 
     # Functions
-    agg = unsupported_function('agg')
```

Comment:
Hm, '`' vs `"`. both are legitimate per PEP 8. Does this show an error?

Reply:
yeah, they standardize quotes to `"` in Black. You are right, both are legitimate in PEP8